spec/models/group_spec.rb

```diff
@@ -13,15 +13,42 @@
     it { is_expected.to validate_presence_of(:leader) }
     it { is_expected.to belong_to(:draw) }
     it { is_expected.to validate_presence_of(:draw) }
-    it { is_expected.to have_many(:members) }
+    it { is_expected.to have_many(:memberships) }
+    it { is_expected.to have_many(:members).through(:memberships) }
   end
 
-  describe '#name' do
-    it "includes the leader's name" do
-      leader = instance_spy('User', name: 'Name')
-      group = FactoryGirl.build_stubbed(:group)
-      allow(group).to receive(:leader).and_return(leader)
-      expect(group.name).to include(leader.name)
+  describe 'size validations' do
+    it 'must be available in the draw' do
+      group = FactoryGirl.build(:group)
+      allow(group.draw).to receive(:suite_sizes).and_return([1])
+      group.size = 2
+      expect(group.valid?).to be_falsey
+    end
+    it 'must not have more members than the size' do
+      group = FactoryGirl.create(:group, size: 1)
```

Comment:
Oh is it because the `member_id`'s are no longer record attributes but come from the `memberships` table?